pkgs/kde/misc/calligra/default.nix

```diff
@@ -0,0 +1,76 @@
+{
+  mkKdeDerivation,
+  lib,
+  fetchurl,
+  boost,
+  eigen,
+  imath,
+  libetonyek,
+  libgit2,
+  libodfgen,
+  librevenge,
+  libvisio,
+  libwpd,
+  libwpg,
+  libwps,
+  okular,
+  perl,
+  pkg-config,
+  poppler,
+  qtkeychain,
+  qtsvg,
+}:
+
+mkKdeDerivation rec {
+  pname = "calligra";
+  version = "4.0.1";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/calligra/calligra-${version}.tar.xz";
+    hash = "sha256-1AH15z9PG9wLNUjqGlCwrBd4we3jCmozWUTtf72I2V8=";
+  };
+
+  extraBuildInputs = [
+    boost
+    qtkeychain
+    qtsvg
+    eigen
+    imath
+    libgit2
+    okular
+    libetonyek
+    libodfgen
+    librevenge
+    libvisio
+    libwpd
+    libwpg
+    libwps
+    poppler
+  ];
+
+  extraNativeBuildInputs = [
+    perl
+    pkg-config
+  ];
+
+  # Recommended by the upstream packaging instructions. RELEASE_BUILD disables
+  # unmaintained components, like Braindump, from being built, and KDE_NO_DEBUG_OUTPUT
+  # is supposed to improve performance in the finished package.
+  extraCmakeFlags = [
+    "-DRELEASE_BUILD=true"
+    ''-DCMAKE_CXX_FLAGS="-DKDE_NO_DEBUG_OUTPUT"''
```

Comment:
Is this an upstream recommendation? I feel like debug logs would be pretty useful when dealing with something as not-very-stable as this.

Reply:
From `README.PACKAGERS`:

> For alpha and beta packages, please build with debug output enabled, but for
production packages the -DCMAKE_CXX_FLAGS="-DKDE_NO_DEBUG_OUTPUT" is recommended.
A significant performance increase will be the result.
>
> All modules may not be ready for production release.
It is recommended to specify -DRELEASE_BUILD=true to exclude these modules.

Reply:
Resolved on the KDE matrix chat. No overrides.